### PR TITLE
add testing for CookieService

### PR DESCRIPTION
**Pull request by ashleygranitto**
This pr features Jasmine tests for the Cookie Service. Additionally, it includes some minor changes to CookieService.js that add ease of testing while maintaining the original functionality.

### CODE REVIEW

File: ui/jasmine_test/spec/cookie-service.spec.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+    // Create a cookie with the example user information
+    cookieService.makeUserInfoCookie(userInfo);
+
+    it('returns a value if a cookie for a given key exists', async () => {
+        const value = cookieService.getCookieValue('userId');
+        expect(value).toEqual('291192');
+    });
+
+    it('returns an empty string if a cookie for a given key does not exist', async () => {
```
Comment (mmallett) [inline ui/jasmine_test/spec/cookie-service.spec.js:37]: not async

File: ui/jasmine_test/spec/cookie-service.spec.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        userId: '291192',
+        gender: 'Female',
+        userAge: 20,
+        deviceType: 'Pixelbook'
+    }
+
+    // Create a cookie with the example user information
+    cookieService.makeUserInfoCookie(userInfo);
+
+    it('returns a value if a cookie for a given key exists', async () => {
```
Comment (mmallett) [inline ui/jasmine_test/spec/cookie-service.spec.js:32]: not async

File: ui/jasmine_test/spec/cookie-service.spec.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {CookieService} from '../../src/utils/CookieService';
+
+describe('Testing that the Cookie Service getCookieValue()', () => {
+    
+    const cookieService = new CookieService();
```
Comment (mmallett) [inline ui/jasmine_test/spec/cookie-service.spec.js:21]: You could combine the `describe` into one, create `cookieService` and call `makeUserInfoCookie` in a `beforeEach`. This setup is common between both test blocks

File: ui/jasmine_test/spec/cookie-service.spec.js (hunk 1 above)
Comment (ashleygranitto) [reply ui/jasmine_test/spec/cookie-service.spec.js:37]: removed

File: ui/jasmine_test/spec/cookie-service.spec.js (hunk 2 above)
Comment (ashleygranitto) [reply ui/jasmine_test/spec/cookie-service.spec.js:32]: removed

File: ui/jasmine_test/spec/cookie-service.spec.js (hunk 3 above)
Comment (ashleygranitto) [reply ui/jasmine_test/spec/cookie-service.spec.js:21]: moved all to one `describe` -- the set up is only required once so I chose not to use the `beforeEach`

**Pull request merged**